generators/app/templates/streamlit/tests/streamlit_app/test_pages.py

```diff
@@ -0,0 +1,22 @@
+"""Test that each Streamlit page can be imported and run."""
+from importlib import import_module
+from pathlib import Path
+
+import pytest
+
+PROJECT_ROOT_PATH = Path(__file__).parent.parent.parent
```

Comment:
It would be useful/cleaner to have that constant in a `src/constants.py` file, no?